utils/awesome-client

```diff
@@ -44,14 +44,16 @@ a_dbus_send()
     $DBUS_SEND --dest=$DBUS_DEST --type=method_call --print-reply \
```

Comment:
commit message: with "when sending a command that it for from stdin failed": s/for/got/ ?